build/gulp/tasks/test-projects.tsx

```diff
@@ -16,11 +17,33 @@ const log = (context: string) => (message: string) => {
   console.log('='.repeat(80))
 }
 
-export const publishPackage = async () => {
-  const filename = tmp.tmpNameSync({ prefix: 'stardust-', postfix: '.tgz' })
-  await sh(`yarn pack --filename ${filename}`)
+const installStardustPackages = async (tmpDirectory: string) => {
```

Comment:
we might name the input parameter as `testProjectDir` - this will be more descriptive and better reflect the expectations of this function. Also, it will make these lines easier to reason about:

```js
 const packageJsonName = path.resolve(tmpDirectory, 'package.json')
 const packageJson = require(packageJsonName)
```

vs 

```js
 const testProjectPackageJson = require(path.resolve(testProjectDir, 'package.json'))
```

Reply:
Used ``testProjectDir` as the param name 👍